lib/src/widgets/custom_image_crop_widget.dart

```diff
@@ -294,7 +295,107 @@ class _CustomImageCropState extends State<CustomImageCrop>
   void onMoveUpdate(MoveEvent event) {
     if (!widget.canMove) return;
 
-    addTransition(CropImageData(x: event.delta.dx, y: event.delta.dy));
+    widget.cropController.addTransition(CropImageData(x: event.delta.dx, y: event.delta.dy));
+  }
+
+  Rect _getInitialImageRect() {
+    assert(_imageAsUIImage != null);
+    final image = _imageAsUIImage!;
+    final cropFitParams = calculateCropFitParams(
+      cropPercentage: widget.cropPercentage,
+      imageFit: widget.imageFit,
+      imageHeight: image.height,
+      imageWidth: image.width,
+      screenHeight: _height,
+      screenWidth: _width,
+      aspectRatio: (widget.ratio?.width ?? 1) / (widget.ratio?.height ?? 1),
+    );
+    final initialWidth = _imageAsUIImage!.width * cropFitParams.additionalScale;
+    final initialHeight = _imageAsUIImage!.height * cropFitParams.additionalScale;
+    return Rect.fromLTWH(
+      (_width - initialWidth) / 2,
+      (_height - initialHeight) / 2,
+      initialWidth,
+      initialHeight,
+    );
+  }
+
+   // just support 'pi/2 * n' angle for now
+  bool _isAngleSupported() {
+    final angle = data.angle;
+    return angle % (pi/2) == 0;
+  }
+
+  void _correctTransition(CropImageData transition, VoidCallback callback) {
+    if (_imageAsUIImage == null ||
```

Comment:
Also add an option in the widget `forceInsideCropArea` and disable it by default so it's not a breaking change + there is no unexpected behaviour.

Reply:
Also add this argument/property to the readme